cpr/ssl_options.cpp

```diff
@@ -0,0 +1,12 @@
+#include "cpr/ssl_options.h"
+
+namespace cpr {
+
+VerifySsl::VerifySsl(bool verify)
+          : verify_{verify} {}
```

Comment:
I ran clang-format over the offending files and resubmitted; I assume that's what you wanted.